<launch>
  <arg name="project_path" value="$(env PROJECT_PATH)"/>
  <arg name="log_dir" default="$(arg project_path)/data" />
  <arg name="topic_description" default="$(find aqua_diayn)/launch/topic_description/back2flipper_topics.yml"/>
  <arg name="task_description" default="$(find aqua_diayn)/launch/task_description/knife_edge.yml"/>
  <!-- <arg name="task_name"/> -->

  <include file="$(find aqua_diayn)/launch/aqua.launch"></include>

  <include file="$(find aqua_diayn)/launch/basic.launch">
    <arg name="topic_description" value="$(arg topic_description)"/>
  </include>

  <node
    name="env_spaces"
    pkg="aqua_diayn"
    type="env_spaces.py"
    output="screen">
      <rosparam command="load" file="$(arg topic_description)"/>
  </node>

  <node
    name="task_manager"
    pkg="aqua_diayn"
    type="task_manager.py"
    output="screen">
      <rosparam command="load" file="$(arg task_description)"/>
  </node>

    <node
      name="baseline_learning"
      pkg="aqua_diayn"
      type="baseline_learning.py"
      args="--log_dir=$(arg log_dir)"
      output="screen">
    </node>

</launch>
